packages/manager/src/features/CloudPulse/Alerts/CreateAlert/Criteria/DimensionFilterValue/ValueSchemas.ts

```diff
@@ -19,16 +18,16 @@ import {
   PORTS_TRAILING_COMMA_ERROR_MESSAGE,
 } from '../../../constants';
 
+const lengthErrorMsg = 'Value must be 100 characters or less.';
```

Comment:
```suggestion
const LENGTH_ERROR_MESSAGE = 'Value must be 100 characters or less.';
```

lets keep the naving convention for contants